internal/gps/vcs_repo_test.go

```diff
@@ -46,6 +52,53 @@ func TestErrs(t *testing.T) {
 	}
 }
 
+func TestNewCtxRepo(t *testing.T) {
+	t.Parallel()
+
+	tempDir, err := ioutil.TempDir("", "go-ctx-repo-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		err = os.RemoveAll(tempDir)
+		if err != nil {
+			t.Error(err)
+		}
+	}()
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	src := filepath.Join(cwd, "_testdata", "src", "corrupt", "corrupt_dot_git_directory.tar")
```

Comment:
Let's not put it in the `src` directory, actually - I realize this is totally not documented, but all the dirs in there are basically fodder for `ListPackages()`. Instead, let's put it in `internal/gps/_testdata/badrepo/corrupt_dot_git_directory.tar`

Also - for our future selves' sake, let's add a `internal/gps/_testdata/badrepo/README.md` that outlines the purpose of the directory (to hold various forms of malformed/corrupted repo archives), and for each archive file (so for now, just one) gives a summary of the way in which each archived repo is broken.